### DISCUSSION TITLE
Clarification for a comment in README

### DISCUSSION THREAD

**Discussion by VlaDexa** [nvim-treesitter/nvim-treesitter#6865] [Q&A] [answered]
```lua
require'nvim-treesitter.configs'.setup {
  -- A list of parser names, or "all" (the five listed parsers should always be installed)
  ensure_installed = { "c", "lua", "vim", "vimdoc", "query" },
```

What exactly does `the five listed parsers should always be installed` mean? Are they a requirement for the user to install, or just means that `nvim-treesitter` will keep them installed?

**Comment by clason** [comment, marked as answer]
> Are they a requirement for the user to install

That. Things will break _badly_ if you don't. (You need to add the markdown parsers these days.)
